app/assets/javascripts/discourse/app/lib/click-track.js

```diff
@@ -11,27 +11,33 @@ import { selectedText } from "discourse/lib/utilities";
 import { wantsNewWindow } from "discourse/lib/intercept-click";
 
 export function isValidLink($link) {
+  const link = $link[0];
+
   // .hashtag == category/tag link
   // .back == quote back ^ button
-  if ($link.is(".lightbox, .no-track-link, .hashtag, .back")) {
+  if (
+    ["lightbox", "no-track-link", "hashtag", "back"].some((name) =>
+      link.classList.contains(name)
+    )
+  ) {
     return false;
   }
 
   if ($link.parents("aside.quote, .elided, .expanded-embed").length !== 0) {
```

Comment:
@jjaffeux what do we want to do with `$.parents()`? Add our own implementation to `dom-utils`?

Reply:
How about

```suggestion
  if (link.closest("aside.quote, .elided, .expanded-embed")) {
```

Reply:
The difference between the two is:

`closest()` "Begins with the current element"
`parents()` "Begins with the parent element"

So for it to behave exactly the same, I guess I'd have to add an extra check:

```suggestion
  const closest = link.closest("aside.quote, .elided, .expanded-embed");
  if (closest && closest !== link) {
```